crates/ruma-client-api/src/discovery/discover_support.rs

```diff
@@ -112,6 +112,13 @@ pub enum ContactRole {
     /// A role intended for sensitive requests.
     Security,
 
+    /// A role for moderation-related queries according to MSC4121.
+    ///
+    /// The future prefix for this if accepted will be `m.role.moderator`
+    #[cfg(feature = "unstable-msc4121")]
+    #[ruma_enum(rename = "support.feline.msc4121.role.moderator")]
```

Comment:
Could you also add `alias = "m.role.moderator"`? That way from the client side it will be deserializable from both the stable and unstable string.